src/case_encoder.h

```diff
@@ -15,6 +15,11 @@
 #ifndef NORMALIZER_CASE_ENCODER_H_
 #define NORMALIZER_CASE_ENCODER_H_
 
+#ifdef _MSC_VER
+#define _REGEX_MAX_STACK_COUNT 200000
+#define _REGEX_MAX_COMPLEXITY_COUNT 0
```

Comment:
Would be great to have a comment here similar to that in the point no. 2 from the PR description.

Reply:
Added comment. Thanks!